hack/godep-save.sh

```diff
@@ -56,7 +56,13 @@ REQUIRED_BINS=(
 )
 
 TMPGOPATH=`mktemp -d`
-trap "rm -rf $TMPGOPATH" EXIT
+TMPCMDREPOS=`mktemp -d`
```

Comment:
We put temporary dirs under `${BASETMPDIR}`:

    TMPGOPATH="$( mktemp --directory --tmpdir="${BASETMPDIR}" )"